test/e2e/specs/editor/editor__post-basic-flow.ts

```diff
@@ -75,15 +75,54 @@ describe( DataHelper.createSuiteTitle( 'Editor: Basic Post Flow' ), function ()
 		it( 'Add post tag', async function () {
 			await editorPage.addTag( tag );
 		} );
+
+		afterAll( async function () {
+			// For mobile, but doesn't hurt to do this for Desktop either.
+			await editorPage.closeSettings();
+		} );
 	} );
 
-	describe( 'Preview', function () {
-		let previewPage: Page;
+	describe( 'Jetpack features', function () {
+		it( 'Open Jetpack settings', async function () {
+			await editorPage.openSettings( 'Jetpack' );
+		} );
+
+		skipItIf( envVariables.TEST_ON_ATOMIC !== true )(
+			'Enter SEO title and preview',
+			async function () {
+				await editorPage.enterSEODetails( {
+					title: 'SEO example title',
+					description: 'SEO example description',
+				} );
+			}
+		);
+
+		it( 'Open social preview', async function () {
+			await editorPage.expandSection( 'Social Previews' );
+			await editorPage.clickSidebarButton( 'Open Social Previews' );
+		} );
 
-		// This step is required on mobile, but doesn't hurt anything on desktop, so avoiding conditional.
-		it( 'Close settings sidebar', async function () {
+		it( 'Show social preview for Tumblr', async function () {
+			// Action implemented as "raw" calls for now (2023-09).
+			const editorParent = await editorPage.getEditorParent();
+			const dialog = editorParent.getByRole( 'dialog' );
+
+			await dialog.getByRole( 'tab', { name: 'Tumblr' } ).click();
+			await dialog.getByRole( 'tabpanel', { name: 'Tumblr' } ).waitFor();
```

Comment:
**Suggestion (non-blocking)**: What do you think of going one small step further and actually validating the content of the preview? If we've added an SEO title, we can look for that text in there, otherwise we can look for the post title. I think that would be a nice final check, and I think we could do it pretty stably!

Reply:
Sure, let me take a look at doing that - I initially chose not to check the rendered preview because it could potentially not generate, etc - and after being burned by the AI Assistant block, I was wary of testing implementation details 😆